packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -15,20 +15,25 @@ import {
 import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
 import { gridClasses } from '../../gridClasses';
 
-interface GridExportFormat {
-  format: ExportTypes;
-  formatOptions?: GridCsvExportOptions;
-}
-
-type GridExportFormatOption = GridExportFormat;
+type GridExportDisplayOptions = {
```

Comment:
Prefer interfaces always.

```suggestion
interface GridExportDisplayOptions {
```

Same for `GridExportOption`.